SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -34,11 +37,20 @@ $DefaultNoteTypeMap = @{
     #'visa' = ''        # Possibly deprecated
     'Wi-Fi Password'    = 'wifi'
 } 
+
+$LoggedOutMessage = 'Error: You don''t apprear to be logged in. Perhaps you need to login with `lpass login`.'
```

Comment:
@TylerLeonhardt Why remove it from `$lpassMessage` ?
Is it a nit ? 

I thought it was cozy within the other `$lpassMessage`

Reply:
I changed the message rather than using the one LastPass had so I pulled it out of the hashtable.

Reply:
@TylerLeonhardt  I kind of liked the old behavior better.
Is this a choice to force the user to login ? 

There's 2 reasons why this message would appear. 
- You are logged out.
- You are logged in but the "active" window closed. 

If you login with lpass --trust email@example.com, you never are really logged out. 

With the old behavior, instead of doing a `lpass status`, the `lpass show` was called directly, which in turn, presented the user with the password prompt. 

So this was kind of a 2 in one compared to now. 
Before: You call `Get-Secret` / `Set-Secret` and the prompt window shows up if your logged in with a disconnected session. 

Currently: You get an error message stating you are not logged in, need to explicitely login again then call the `Get/Set-Secret` cmdlet a second time.

In both scenario, if you are already logged out, then you need to login anyway but 99.9% of the time, I just need to re-establish the session (prompt) and not actually login again.

Reply:
tl;dr 

When not connected (but logged in) 
**Old flow** 
- User type:  Get-Secret
- User prompted:  Password prompt
- Secret is returned

**Current flow**
- User type: Get-Secret ... 
-  Message: Error: You are currently not logged in ... Try login in with lpass login. 
- User type: lpass login
- User type: Get-Secret ... 
- Secret is returned

Considering the password prompt is to be entered hourly, the second flow become more annoying (You kind of always want to have the login command not far because you'll likely need it). The first (old flow) just prompt you directly for your password and continue afterward. 

In both scenarios, if you are truly logged out, you'll see the error stating you need to login.

Reply:
@itfranck I agree with you. Take a look at my last commit